tests/test_batchuploader.py

```diff
@@ -47,19 +48,19 @@ def test_writer_maxitemsize(self):
             ValueTooLarge,
             'Value exceeds max encoded size of 1048576 bytes:'
             ' \'{"b+\\.\\.\\.\'',
-            w.write, {'b'*(m/2): 'x'*(m/2)})
+            w.write, {'b'*int(m/2): 'x'*int(m/2)})
```

Comment:
interesting, I was taught to do this differently :) and linters usually show an error for such formatting. I don't have a strong opinion here, "Use your own judgment" can be applied.